<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolsWidget</class>
 <widget class="QWidget" name="toolsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="buttonExtractText">
     <property name="text">
      <string>Detect and Extract Text</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonDetectEdges">
     <property name="text">
      <string>Detect Edges</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonAlignImage">
     <property name="text">
      <string>Align Image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonDetectFaces">
     <property name="text">
      <string>Detect Faces</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>155</width>
       <height>518</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
